.github/workflows/changelog-to-confluence.yaml

```
@@ -0,0 +1,31 @@
+name: Publish Changelog to Confluence
+on:
+    pull_request:
```

nit: in the current configuration, the Github Action will be triggered on every commit (this includes drafts, updates, rebases, etc.), which might cause noise or undesired content pulled from the changelog. 

You could trigger on every push instead, so it would run only once when the PR gets merged. 

```suggestion
    push:
```